GTBitsOfGood/canine-assistants
# added image upload button to dog forms

## Added Image Upload to Add Dog and Edit Dog Forms

Issue Number(s): #108

What does this PR change and why? PR adds image upload button to add-dog and edit-dog forms for the ability to add and manage profile pictures of dogs

### Checklist


### Critical Changes

- None

### Related PRs

PR Number(s): #107 

### How To Test

Click Add Dog on dashboard. Click Upload Image. A preview will appear. If an image is already available for the dog, a button for Replace Image and Delete will show.
